itou/templates/employee_record/list.html

```diff
@@ -135,6 +143,20 @@ <h5>Nous transférons vos fiches salarié à l'ASP afin de vous faire gagner du
                     </div>
                 {% endif %}
 
+                <div class="row justify-content-end">
+                    <div class="col-12 col-sm-auto">
+                        <span class="font-weight-bold mr-2">Trier par</span>
+                        <button type="button" class="btn btn-outline-primary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            {{ ordered_by_label }}
+                        </button>
+                        <div class="dropdown-menu dropdown-menu-right" id="order-form-group">
+                            {% for order_value, order_label in form.order.field.choices %}
+                                <button class="dropdown-item {% if order_value == form.order.value %}active{% endif %}" type="button" value={{ order_value }}>{{ order_label }}
```

Comment:
Si on ajoute une option avec un espace.
```suggestion
                                <button class="dropdown-item {% if order_value == form.order.value %}active{% endif %}" type="button" value="{{ order_value }}">{{ order_label }}
```